runners-msi/common/common-directories.wxi

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It would be tricky, as the one original directory was split into two new ones. Is listing them a help instead? I went for two distinct subdirs, to make splitting easier.

The only changes to wxi's specifically are removing the TeamCity addin from `runner-features.wxi` and `runner-files.wxi`.

The only real Wix changes in general are the above, and the merge of `runners.wxs` and `nunit.wxs`.

In general, there were previously three installers. NUnit (everything), Runners (console/engine) and Framework. Framework is untouched, NUnit is removed, and Runners replaces NUnit, with the framework and TC extension removed.